### PR TITLE
Update schema/snapcraft.yaml

### PR DESCRIPTION
**Pull request by diddledan**
Improve compatibility with yaml to json converters

Signed-off-by: Daniel Llewellyn <daniel@bowlhat.net>

- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [ ] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

**Comment by diddledan** [comment]
hmm, don't merge - I think I screwed up

**Comment by diddledan** [comment]
fixeded it :-)

File: schema/snapcraft.yaml [hunk 1]
```diff
@@ -224,10 +224,10 @@ properties:
   apps:
     type: object
     additionalProperties: false
-    validation-failure:
-            "{!r} is not a valid app name. App names consist of upper- and
+    validation-failure: |
```
Comment (sergiusens) [inline schema/snapcraft.yaml]: by using | you are saying, use this string verbatim.

Comment (diddledan) [reply schema/snapcraft.yaml]: perhaps a better choice would be `>`.  ???

Comment (sergiusens) [reply schema/snapcraft.yaml]: yes, but let's go with the json PR

**Comment by sergiusens** [comment]
Closing in favour of #2448

**Pull request closed without merging**